.github/workflows/verify.yml

```diff
@@ -1,23 +1,23 @@
+---
 name: verify
 on:
   pull_request:
     branches:
       - 'master'
   push:
-    branches:
-      - 'master'
-      - pr-*
```

Comment:
keep master or it will trigger 2x workflow when branch is not in sync with master